docs/content/integrations/dbt/reference.mdx

```diff
@@ -159,6 +159,56 @@ In your CI/CD workflows for your Dagster and dbt project:
 
 ---
 
+## Leveraging dbt defer with branch deployments
+
+It is possible to leverage [dbt defer](https://docs.getdbt.com/reference/node-selection/defer) by passing a `state_path` to <PyObject object="DbtProject" module="dagster_dbt" />. This allows you to run a subset of models or tests based on the changes made since the state passed to `state_path`.
+
+In practice, this is most useful when combined with branch deployments in Dagster+, to test changes made in your branches. This can be easily done by updating your Dagster code and your CI/CD files.
+
+First, update your Dagster code to pass a `state_path` to your `DbtProject` object. Also, update the dbt command in your `@dbt_assets` definition to pass the defer args using `get_defer_args`.
+
+```python startafter=start_use_dbt_defer_with_dbt_project endbefore=end_use_dbt_defer_with_dbt_project file=/integrations/dbt/dbt.py dedent=4
+import os
+from pathlib import Path
+
+from dagster import AssetExecutionContext
+from dagster_dbt import DbtCliResource, DbtProject, dbt_assets
+
+my_dbt_project = DbtProject(
+    project_dir=Path(__file__).joinpath("..", "..", "..").resolve(),
+    packaged_project_dir=Path(__file__)
+    .joinpath("..", "..", "dbt-project")
+    .resolve(),
+    state_path=Path("state"),
+)
+my_dbt_project.prepare_if_dev()
+
+@dbt_assets(manifest=my_dbt_project.manifest_path)
+def my_dbt_assets(
+    context: AssetExecutionContext,
+    dbt: DbtCliResource,
+):
+    yield from dbt.cli(["build", *dbt.get_defer_args()], context=context).stream()
+```
+
+Note that value passed to `state_path` must be a path, relative to the dbt project directory, to a directory of dbt artifacts. In the code above, we set the `state_path` to 'state/'.
+
+Now that the Dagster code is updated, we need to update your CI/CD files to manage the state of your dbt project using the dagster-cloud CLI. This will ensure that dbt correctly uses your deferred artifacts as part of our Dagster+ branch deployments.
+
+You might have one or two CI/CD files to manage your production and branch deployments. In these files, find the steps related to your dbt project - refer to the [Deploying a Dagster project with a dbt project](#deploying-a-dagster-project-with-a-dbt-project) section for more information.
+
+Once your dbt steps are located, add the following step to manage the state of your dbt project.
+
+```bash
+dagster-cloud ci dagster-dbt project manage-state --file path/to/project.py
```

Comment:
Does this command store the prod manifest at the path `state` used earlier in the code example?

Reply:
Yes. It is mentioned below, but I can rephrase if you think it should be clearer.

Reply:
I think we should lead with this and then do the code change next. It would help to have an overview section that talks through the steps first briefly. For example:


In order to be able to use `dbt defer` we will do the following:
1. Add a command to our CI/CD pipeline that saves a state file to a given location
2. Update our Dagster dbt code to reference this new state file. 
3. ...Some other step?